modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/allow_list_sync_job.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class AllowListSyncJob
+    include Sidekiq::Job
```

Comment:
How do you plan to call this Sidekiq job? I don't see it called or added to periodic_jobs.

Reply:
We haven't installed it in `periodic_jobs.rb` yet, but my thought was once every 15 minutes. Would this likely be rate limited?